pkg/bindinfo/binding_plan_generation.go

```diff
@@ -80,6 +81,15 @@ func (g *planGenerator) Generate(defaultSchema, sql, charset, collation string)
 	return
 }
 
+type tableName struct {
+	schema string
+	name   string
```

Comment:
We can try ```unique.Handle[string]```

https://pkg.go.dev/unique#Handle

Reply:
Emm, since in this scenario "EXPLAIN EXPLORE", performance is not the first priority, then we'd tend to keep our code simpler.

Reply:
Yes... but they all are not exposed... and since these structures are simple and only used internal, this seems OK to re-define them.

Reply:
https://github.com/pingcap/tidb/pull/60230

We actually have one that is being done, but it is really a hard labor